packages/itwinui-css/backstop/tests/color.html

```diff
@@ -130,6 +131,7 @@ <h2 class="iui-text-title iui-text-spacing">Background</h2>
           <ul>
             <li class="iui-text-block">Normal text</li>
             <li class="iui-text-block iui-text-muted">Muted text</li>
+            <li class="demo-text-placeholder">Placeholder text</li>
```

Comment:
Do we also have to add this line near L186, under "Background backdrop"?

Reply:
I think placeholder text color would only ever appear above `--iui-color-background` and never above any other background color.

Reply:
In the dev documentation I put a brief explanation.  Without me expanding upon what is already written, [does this section make sense to you?](https://dev-itwinui.bentley.com/docs/variables#background)  Because if this isn't clear enough, I suppose it should be updated.

Reply:
Thanks! Yes, the description was clear for me.

Although not needed, it would help if we have some picture examples for each color. E.g., a screenshot (or codesandbox type example) of a layout that uses most/all colors, along with arrows to show which color is used where.

E.g., for this [layout](https://itwin.github.io/iTwinUI-layouts), we can have arrows to the tiles to say it is `background` and to the background of the page to say it is `background-backdrop`